kore/test/Test/Kore/IndexedModule/Resolvers.hs

```diff
@@ -285,6 +295,11 @@ test_resolvers =
                 (getSentenceSymbolOrAliasHead objectB [])
             )
         )
+    , testCase "sort indexed axioms"
```

Comment:
You have to do a lot of digging to see what axioms are sorted *by*.

Reply:
Though I suppose it doesn't actually matter, so long as the order is always the same. Still.